File: MojStomatolog/UI/mojstomatolog_mobile/lib/models/working_hours.dart
import 'package:json_annotation/json_annotation.dart';

part 'working_hours.g.dart';

@JsonSerializable()
class WorkingHours {
  int? id;
  int? dayOfWeek;
  String? startTime;
  String? endTime;
  String? breakStartTime;
  String? breakEndTime;
  int? userModifiedId;

  WorkingHours();

  factory WorkingHours.fromJson(Map<String, dynamic> json) =>
      _$WorkingHoursFromJson(json);
  Map<String, dynamic> toJson() => _$WorkingHoursToJson(this);
}

File: MojStomatolog/MojStomatolog.Models/Core/WorkingHours.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MojStomatolog.Models.Core
{
    public class WorkingHours
    {
        [Key]
        public int Id { get; set; }
        public DayOfWeek DayOfWeek { get; set; }
        public TimeSpan StartTime { get; set; }
        public TimeSpan EndTime { get; set; }
        public TimeSpan BreakStartTime { get; set; }
        public TimeSpan BreakEndTime { get; set; }

        public int? UserModifiedId { get; set; }
        [ForeignKey("UserModifiedId")]
        public User? UserModified { get; set; }
    }
}